### DISCUSSION TITLE
Doubt about the chainconfig

### DISCUSSION THREAD

**Discussion by infocodiste** [Web3Auth/Web3Auth#639] [Help]
what is the displayName and tickerName in chainConfig?
and what will be the value of these 2 variables for polygon mainnet network?
Currently I am using the following, please correct me if I am using it wrong:

displayName: "Polygon"
tickerName: "Polygon"

**Comment by irux** [comment]
I think this two fields are not mandatory as long as you provide an RPC and the correct chain id (in hex). I didn't need it and it worked great!

**Reply by shahbaz17** [reply]
Yes, that's correct @irux!

@infocodiste these are optional fields. But here's how the `chainConfig` would look like for `Polygon`:

```js
chainConfig: {
    chainNamespace: "eip155",
    chainId: "0x89", // hex of 137, polygon mainnet
    rpcTarget: "https://rpc.ankr.com/polygon",
    // Avoid using public rpcTarget in production.
    // Use services like Infura, Quicknode etc
    displayName: "Polygon Mainnet",
    blockExplorer: "https://polygonscan.com",
    ticker: "MATIC",
    tickerName: "Matic",
  },
  ```
